### PR TITLE
feat: markdown for notifications

### PR DESCRIPTION
**Pull request by diogogmatos**
Added compatibility for markdown in the notification descriptions, this markdown is taken as a string and converted to html and then JSX. To achieve this, I installed the following (very light) npm package: [markdown-to-jsx](https://www.npmjs.com/package/markdown-to-jsx).

### CODE REVIEW

**Comment by diogogmatos** [comment]
[feat: markdown for notifications](https://trello.com/c/E8fkrob3/76-feat-markdown-for-notifications)

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Please remove `select-none` from notifications

File: components/Notifications/Notifications.tsx [hunk 1]
```diff
@@ -2,6 +2,10 @@ import React from "react";
 
 import moment from "moment-timezone";
 
+import Markdown from "markdown-to-jsx";
+
+import { render } from "react-dom";
```
Comment (ruilopesm) [inline components/Notifications/Notifications.tsx:7]: What is this import being used for?

Comment (diogogmatos) [reply components/Notifications/Notifications.tsx:7]: nothing.

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**